index.js

```diff
@@ -0,0 +1,43 @@
+// IMPORT MODULES
+import StorageBooks from './modules/storageBooks.js';
+import toggleVisbility from './modules/toggleVisibility.js';
+import theClasses from './modules/theClasses.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+// import { DateTime } from "luxon";
+
+// CREATE ELEMENTS
+const addBook = document.getElementById('add-book');
+const allBooksSection = document.getElementById('all-books');
+const addNewBookSection = document.getElementById('add-new-book');
+const contactSection = document.getElementById('contact');
+const listLink = document.getElementById('list-link');
+const addLink = document.getElementById('add-link');
+const contactLink = document.getElementById('contact-link');
+
+// SET CURRENT TIME
+document.getElementById('date').innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
```

Comment:
- [Optional] We can also ameliorate this by making sure that time is constantly updated on the view instead of reloading the page.